### PR TITLE
Make this work on windows

### PR DESCRIPTION
**Pull request by codebreach**

### CODE REVIEW

**Comment by kaanon** [comment]
Looks good, I'll test and merge soon.

File: index.js [hunk 1]
```diff
@@ -51,7 +52,7 @@ function handlebars(data, opts) {
 		if (depth > maxDepth) { return; }
 		base = base || dir;
 		fs.readdirSync(dir).forEach(function (basename) {
-			var filename = dir + '/' + basename;
+			var filename = path.join(dir, basename);
```
Comment (AGvin) [inline index.js]: We need to add
``` 
dir = path.normalize(dir);
```
before execute

```
registerPartials(dir, dir, 0);
```

Example
```
	// Go through a partials directory array
	if(options.batch){
		// Allow single string
		if(typeof options.batch === 'string') options.batch = [options.batch];

		options.batch.forEach(function (dir) {
			dir = path.normalize(dir);
			registerPartials(dir, dir, 0);
		});
	}
```

If it willn't be fixed, we will have normalized "filename" and not normalized "basename" variables;

**Comment by AGvin** [comment]
According this changes, now, i can't use nested templates. For example,
if
```
batch: [srcDemoPath + 'templates/partials/'],

```
call partial:
```
{{> footer/js}}
```
partial path:
```
templates/partials/footer/js.hbs
```

**Comment by kaanon** [comment]
Merged in changes in https://github.com/kaanon/gulp-compile-handlebars/pull/21
Thanks for the contribution.

**Pull request closed without merging**